src/background/index.mjs

```
@@ -276,36 +412,98 @@
-      enabled: true,
-    })
+    try {
+      if (!tab.url || !info.url) { // Check if tab.url or info.url is present, as onUpdated can fire for various reasons
+        // console.debug('[background] onUpdated event without URL, skipping side panel update for tab:', tabId, info);
+        return
+      }
+      console.debug(
+        `[background] tabs.onUpdated event for tabId: ${tabId}, status: ${info.status}, url: ${tab.url}`,
+      )
+      // eslint-disable-next-line no-undef
+      if (chrome && chrome.sidePanel) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)